config/capsule-answers.yaml

```diff
@@ -0,0 +1,5 @@
+---
+certs:
+  generate: false
+  deploy: true
+capsule: true
```

Comment:
Why do we have capsule-certs-generate include puppet=true, if we don't feel that the majority of installations would be including it?  Would not including puppet affect pulp's puppet support?  (I assume not, but curious.)